.github/workflows/add-orbit-chain.yml

```diff
@@ -78,8 +78,8 @@ jobs:
 
       - name: Commit changes
         run: |
-          git add packages/arb-token-bridge-ui/src/**/*
-          git add packages/arb-token-bridge-ui/public/**/*
+          git add "packages/arb-token-bridge-ui/src/**/*"
+          git add "packages/arb-token-bridge-ui/public/**/*" ":!public/images/__auto-generated/**"
```

Comment:
Do not commit auto generated files (we can't anyway because it's in git ignore, would throw errors)